src/home_robot/home_robot/utils/point_cloud_torch.py

```diff
@@ -34,6 +37,59 @@ def depth_to_xyz(depth: torch.Tensor, camera: Camera):
     return xyz
 
 
+def unproject_masked_depth_to_xyz_coordinates(
```

Comment:
Yeah it could share a codepath. This implementation came from utils/point_clouds.py, but it was written in torch so I moved it here. 
I like this implementation because it's batched, and uses K_inv (so works with other camera models besides pinhole). It might be marginally slower, but that's not a problem.
Can I just add a TODO to this?